.circleci/build.sh

```diff
@@ -34,7 +34,7 @@ pip install ninja
 pip install lark-parser
 
 # Install Pytorch
-patch -p1 < xla/pytorch.patch
+xla/scripts/apply_patches.sh
```

Comment:
Doesn't `kokoro/ubuntu/common.sh` need the same change?

Reply:
Yeah